workers/unity/Packages/io.improbable.gdk.core/Subscriptions/Callbacks/IndexedCallbacks.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+
+namespace Improbable.Gdk.Subscriptions
+{
+    // Slight adjustment to the component callbacks class
+    // todo check if this can have in params for ops (i.e. can it be used with a non readonly struct)
```

Comment:
I know it wasn't you, but we should create a ticket for this todo and mention it here (or remove the todo). Otherwise it will never be done